### ISSUE TITLE
mcmcp demo info handling fails in long chains

### ISSUE THREAD

**Issue by jcpeterson** [Dallinger/Dallinger#570]
Solution: sort infos on the frontend by id rather than by list order

```
            if (resp.infos[0].id > resp.infos[1].id) {
                animal_0 = JSON.parse(resp.infos[0].contents);
                animal_1 = JSON.parse(resp.infos[1].contents);
            } else {
                animal_0 = JSON.parse(resp.infos[1].contents);
                animal_1 = JSON.parse(resp.infos[0].contents);
            }
```

**Comment by suchow** [comment]
Note to selves: this was caused by a non-deterministic sort order of infos returned by the `/node/<node_id>/infos` route, possibly interacting with timing in some way.

**Comment by suchow** [comment]
Closed by #571.
